src/recipe/jinja.rs

```diff
@@ -309,6 +309,7 @@ fn default_tests(env: &mut Environment) {
     env.add_test("string", minijinja::tests::is_string);
     env.add_test("sequence", minijinja::tests::is_sequence);
     env.add_test("boolean", minijinja::tests::is_boolean);
+    env.add_test("startingwith", minijinja::tests::is_startingwith);
```

Comment:
I would then also add `endingwith`. 

Can you also add it to the docs?

And lastly, check in the python test that startingwith evaluates to `true` (or false) in your test.

Reply:
> check in the python test that startingwith evaluates to true (or false) in your test

happens through snapshots

Reply:
But there is no updated snapshot :)

Reply:
<img width="843" alt="image" src="https://github.com/user-attachments/assets/01f620a9-8ed5-4094-97f1-d7941a8bc603" />